File: Keyboard_Row/Keyboard_Row.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	words := []string{"Hello", "Alaska", "Dad", "Peace"}
	fmt.Println(findWords(words))
}

func findWords(words []string) []string {

	key := []string{"qwertyuiop", "asdfghjkl", "zxcvbnm"}
	var res []string
	for i := 0; i < len(words); i++ {
		var str string
		l := words[i]
		for _, k := range key {
			for _, char := range words[i] {
				if strings.Contains(k, strings.ToLower(string(char))) {
					str += string(char)
				} else {
					break
				}
			}

		}

		if l == str {
			res = append(res, words[i])
		}

	}
	return res
}

File: Keyboard_Row/Keyboard_Row.rs

fn main(){
    let words:Vec<String> = vec!["Hello".to_string(),"Alaska".to_string(),"Dad".to_string(),"Peace".to_string()];
    println!("{:?} ",findWords(words));
}


fn findWords(words:Vec<String>) -> Vec<String>{
    let keys = ["qwertyuiop","asdfghjkl","zxcvbnm"];
    let mut res = Vec::new();
    for word in words{
        let l = word.len();
        let mut str1 = "".to_string();
        for k in keys{
            for chars in word.chars(){
                let t = k.contains(&*&chars.to_string().to_lowercase());
                if t{
                    str1+=&chars.to_string().to_lowercase();
                }else{
                    break
                }

            }
        }
        if l == str1.len(){
            res.push(word);
        }
    }
    return res
}